playbooks/adhoc/uninstall.yml

```diff
@@ -236,3 +239,5 @@
   tasks:
     - name: restart docker
       service: name=docker state=restarted
+    - name: restart NetworkManager
+      service: name=NetworkManager state=restarted
```

Comment:
Is a restart needed here, or is it possible to reload/reconfigure this through nmcli?